server/src/services/commits.ts

```diff
@@ -74,21 +74,18 @@ const addCommit = async (
   commitData: CommitRequest
 ): Promise<CommitResponse> => {
   // Make sure that listing and customer exists
```

Comment:
Not very important but can I think it's better to change this comment to `// Check ...` to standardise with the other check comments.
Also, to standardise, should add full stop after the comment.

Reply:
@karenfrilya97 Ok will change to "Check"! But for fullstop wise I think normally our `// ...` comments don't have (is more like a phrase) but `/** ... */` comments have